paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsViewModelTestInjection.kt

```diff
@@ -0,0 +1,89 @@
+package com.stripe.android.paymentsheet
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.lifecycle.SavedStateHandle
+import androidx.test.core.app.ApplicationProvider
+import com.stripe.android.core.Logger
+import com.stripe.android.core.injection.Injectable
+import com.stripe.android.core.injection.Injector
+import com.stripe.android.core.injection.InjectorKey
+import com.stripe.android.core.injection.WeakMapInjectorRegistry
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.paymentsheet.analytics.EventReporter
+import com.stripe.android.paymentsheet.injection.PaymentOptionsViewModelSubcomponent
+import com.stripe.android.paymentsheet.viewmodels.BaseSheetViewModel
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import org.junit.After
+import org.junit.Rule
+import org.mockito.kotlin.any
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import javax.inject.Provider
+
+@ExperimentalCoroutinesApi
+internal open class PaymentOptionsViewModelTestInjection {
+    @get:Rule
+    val rule = InstantTaskExecutorRule()
+
+    private val testDispatcher = TestCoroutineDispatcher()
```

Comment:
np: should be `private val testDispatcher = UnconfinedTestDispatcher()`. TestCoroutineDispatcher() was deprecated.